bin/build_perf_tool

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+
+CURRENT_DIR=$("$(dirname "$0")/abspath")
+
+PERF_TEST_IMAGE="juxtaposer"
+PERF_TEST_IMAGE_TAG="$PERF_TEST_IMAGE:latest"
+
+echo "Current dir: $CURRENT_DIR"
+
+pushd "$CURRENT_DIR/juxtaposer"
```

Comment:
This seems like the time to bring up naming questions 😬

- I'm looking at this and `juxtaposer` is just not evocative of what it is that this actually doing. I _think_ you called it `juxtaposer` because it's comparing direct-to-db vs via secretless (eg comparing them side-by-side), but that's not all it's doing - it's also validating long-time performance and validating performance under load.

  Infra has scripts to perf test Conjur, and they call them `exercise` and `pummel` (which is pretty evocative of what they're doing). I wonder if we could come up with a better name here.

- I wonder whether there are better names for the `formatter` and `tester` components - thinking of @jonahx's recent repost of [this link](http://www.carlopescio.com/2011/04/your-coding-conventions-are-hurting-you.html) warning against components with `-er` names.

- Putting this stuff in `bin/` seems odd. I'm wondering if we should be putting the integration tests in `test/integration` and putting this in `test/performance`, or some other place? Just in `performance-testing` maybe?